.gitignore

```diff
@@ -1,9 +1,8 @@
- 
 .DS_Store
 node_modules
 dist
 package-lock.json
 coverage
 .env
 .vscode
-html-report
\ No newline at end of file
+html-report
```

Comment:
Why are we having this here, I thought this file is meant to be inside the coverage folder?

Reply:
Let me fix it